packages/aws-cdk-lib/aws-scheduler/README.md

```diff
@@ -1,27 +1,64 @@
-# AWS::Scheduler Construct Library
+# Amazon Scheduler Construct Library
 
+Amazon EventBridge Scheduler](https://aws.amazon.com/blogs/compute/introducing-amazon-eventbridge-scheduler/) is a feature from Amazon EventBridge that allows you to create, run, and manage scheduled tasks at scale. With EventBridge Scheduler, you can schedule one-time or recurrently tens of millions of tasks across many AWS services without provisioning or managing underlying infrastructure.
```

Comment:
another comment on impending linter errors that i feel should have been enforced somewhere: these paragraphs have to be broken into separate lines with max 150 characters i believe

Reply:
Updated readme. So the rule on characters per line in README is not enforced anywhere? What would be a good linter to add this rule to?

Reply:
it has been enforced in the past, i think our repo restructure may have wreaked havoc on our linter perhaps. i've been meaning to look into it